### DISCUSSION TITLE
Welcome to nomadstudy-github-list Discussions!

### DISCUSSION THREAD

**Discussion by ghost** [Nomad-Study/nomadstudy-github-list#53] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

**Comment by ghost** [comment]
Welcome to Nomad Study! 🥳

**Comment by jiisuu** [comment]
와아아아우 노마드 스터디 어쎔블
